{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 6",
      "fix_state": "Under investigation",
      "package_name": "libX11",
      "cpe": "cpe:/o:redhat:enterprise_linux:6"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Under investigation",
      "package_name": "libX11",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Under investigation",
      "package_name": "libX11",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 9",
      "fix_state": "Under investigation",
      "package_name": "libX11",
      "cpe": "cpe:/o:redhat:enterprise_linux:9"
    }
  ],
  "threat_severity": "Moderate",
  "public_date": "2023-10-04T00:00:00Z",
  "bugzilla": {
    "description": "stack exhaustion from infinite recursion in PutSubImage()",
    "id": "2242253",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2242253"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "",
    "cvss3_scoring_vector": "",
    "status": ""
  },
  "iava": "",
  "cwe": "CWE-400",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2023-43786",
  "document_distribution": "",
  "details": [
    "No description is available for this CVE."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2023-43786\nhttps://nvd.nist.gov/vuln/detail/CVE-2023-43786"
  ]
}